docker/develop/Dockerfile

```diff
@@ -25,16 +25,23 @@ RUN apt-get -y --no-install-recommends install build-essential python-software-p
         # code coverage
         lcov \
         # other
-        wget curl file unzip gdb iputils-ping vim ccache \
+        wget file unzip gdb iputils-ping vim ccache \
         gcovr cppcheck doxygen graphviz graphviz-dev; \
     apt-get -y clean
 
-RUN pip3 install --upgrade pip; \
-    pip3 install flask requests
 
-# install cmake 3.5.1
+# install curl with SSL, 7.58.0
+RUN (cd /tmp; wget  https://curl.haxx.se/download/curl-7.58.0.tar.gz; \
+    tar -xf curl-*.tar.gz; \
+    cd curl-7.58.0; \
+    ./configure --with-ssl; \
+    make -j${PARALLELISM} install); \
+    ldconfig; \
+    rm -rf /tmp/curl*
+
+# install cmake 3.10.2
```

Comment:
Having a greater version can introduce build inconsistencies, because features from versions greater than 3.7 can be used without CI failure. Having 3.7 version in Dockerfile is fine.

Reply:
@lebdron what is the exact cmake version required for SWIG?